docs/antora/modules/ROOT/pages/configuration/index.adoc

```diff
@@ -1,21 +1,29 @@
 = Lisk Service Configuration Reference
 Muhammad Talha <muhammad.talha@lightcurve.io>
-:toc:
+// Settings
+:toc: preamble
+:toclevels: 5
+:page-toclevels: 3
+:idprefix:
+:idseparator: -
+:experimental:
+:source-language: js
+
 :url_microservices: index.adoc#microservices
 :url_intro_services: index.adoc
+
 :url_pm2: https://pm2.keymetrics.io/
-:url_config_file: https://github.com/LiskHQ/lisk-service/blob/development/ecosystem.config.js
+:url_config_file: https://github.com/LiskHQ/lisk-service/blob/release/0.7.0/ecosystem.config.js
```

Comment:
Please add a TODO to update the URL to point to the `main` branch later once we make the release in prod.